### PR TITLE
3 functions map to user input

### PR DESCRIPTION
**Pull request by nohelia-f**
I have an error here because I don't understand how to manage the event listener yet. ):(

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="sft-n-hrd.js"></script>
+    <title>sft n hrd</title>
+    <meta name="description" content="A simple website to play with curious measurements.">
+  </head>
+  <body>
+    <h1>sft n hrd</h1>
+
```
Comment (fogg4444) [inline index.html:12]: all these changes will be present in the other PR. You can merge master into this branch and these changes will do away

File: sft-n-hrd.js [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
 
-// const totalAge = addNumbers(30, 31);
-
-// console.log(totalAge);
-var inchesToCmsConverstionFactor = 2.54
-const brianLenghtInches = 4.5;
-const brianLenghtInchesWhenHard = 6.75;
+enterButton.addEventListener('click', function(){
+  let softLenght = document.getElementById("softbox").value;
+
```
Comment (fogg4444) [inline sft-n-hrd.js:6]: can we put a log right here on line 6? Just to see that this value is being accessed correctly?

**Review by fogg4444 — CHANGES_REQUESTED** [review]

**Comment by nohelia-f** [comment]
Hey @fogg4444 I have conflicts on this branch but I don't understand why... would you help me to take a look at this?

**Comment by nohelia-f** [comment]
Added an event to print the difference between soft and hard inputs on the screen.

image

File: sft-n-hrd.js [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
+enterButton.addEventListener('click', function(){
+  let softLenght = document.getElementById("softbox").value; //Getting info from softbox
+  let hardLenght = document.getElementById("hardbox").value; //Getting info from hardbox
+  console.log(softLenght, hardLenght);
+  document.getElementById("result").innerHTML = (hardLenght - softLenght); //Printing the difference on the screen
+});
+
+/*
+
+//Conversion de pulgadas a centimetros
```
Comment (fogg4444) [inline sft-n-hrd.js:14]: Can we delete all this old code we're not using for now?

**Review by fogg4444 — APPROVED** [review]

File: sft-n-hrd.js (hunk 3 above)
Comment (nohelia-f) [reply sft-n-hrd.js:14]: > Can we delete all this old code we're not using for now?

No, sorry

**Pull request merged**